app/models/provider_interface/provider_user_form.rb

```diff
@@ -0,0 +1,27 @@
+module ProviderInterface
+  class ProviderUserForm < SupportInterface::ProviderUserForm
+    attr_accessor :current_provider_user
+
+    validate :permitted_providers
+
+    def available_providers
+      @available_providers ||= Provider
```

Comment:
Could this be a use case for `ProviderOptionsService`?

Reply:
:+1: Would it be better to move the scope `Provider.with_users_manageable_by(provider_user)` into this service to keep the model lean? This is the only place the scope is used.

Reply:
Amended in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1823/commits/bb5abe8e96c3434943666f0f195f06dbb2cf4211

Reply:
:+1: Nice